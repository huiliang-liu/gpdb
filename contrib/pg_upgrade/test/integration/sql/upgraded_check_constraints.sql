select * from users_with_check_constraints;

insert into users_with_check_constraints values (2, 'Jane');
