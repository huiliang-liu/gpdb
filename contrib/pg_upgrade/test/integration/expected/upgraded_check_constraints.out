select * from users_with_check_constraints;
 id | name 
----+------
 1  | Joe  
(1 row)

insert into users_with_check_constraints values (2, 'Jane');
ERROR:  new row for relation "users_with_check_constraints" violates check constraint "users_with_check_constraints_id_check"
DETAIL:  Failing row contains (2, Jane).
